### PR TITLE
Dockerfile & docker compose for next-app

### PR DESCRIPTION
**Pull request by Ronfflex**

### CODE REVIEW

**Comment by Guiiz94** [comment]
@Ronfflex you push files about the front, just push your changes about docker

File: next-app/Dockerfile
Comment (MTthoas) [inline next-app/Dockerfile:1]: Why do you need of python, make and g++ for a next-app ?

File: docker-compose.yml [hunk 1]
```diff
@@ -51,10 +40,17 @@ services:
     networks:
       - dex-network
 
+  next-app:
+    build:
+      context: ./next-app
+      dockerfile: Dockerfile
+    ports:
+      - "3001:3000"
```
Comment (MTthoas) [inline docker-compose.yml:48]: Why in your docker-cpmpose u define a port redirection from 3000 to 3001 for exposing it at 3000 in your dockerfile?

**Review by MTthoas — CHANGES_REQUESTED** [review]

**Comment by MTthoas** [comment]
> @Ronfflex you push files about the front, just push your changes about docker

i agree

File: docker-compose.yml (hunk 1 above)
Comment (Ronfflex) [reply docker-compose.yml]: My bad

Comment (Ronfflex) [reply docker-compose.yml]: Actually no. That was write but I modified it anyway.

EXTERNAL:INTERNAL

**Comment by Ronfflex** [comment]
> > @Ronfflex you push files about the front, just push your changes about docker
> 
> i agree

I don't since I modified the front files to make the dockerfile compile. Both are related.

**Review by MTthoas — APPROVED** [review]

**Pull request merged**